### PR TITLE
Add check for not supported webassembly.exception.stack

### PR DESCRIPTION
**Pull request by ravisumit33**
Webassembly.Exception.stack is still on supported on latest safari browser (16.2). Although [MDN](https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/Exception) shows supported since 15.2 but its incorrect and I have reported the issue https://github.com/mdn/browser-compat-data/issues/18938. As mentioned in the issue it can be easily verified by running below code:

```js
const tag = new WebAssembly.Tag({ parameters: ["i32"] });

function throw_exception_with_stacktrace(param) {
    return new WebAssembly.Exception(tag, [param], { traceStack: true });
}

const ex = throw_exception_with_stacktrace(42);
console.log(ex.stack);
```
Added a safety check in this PR for cases like this where stack comes undefined.

### CODE REVIEW

**Comment by ravisumit33** [comment]
@sbc100 This is the new PR which I created to resolve the issue we faced in https://github.com/emscripten-core/emscripten/pull/18800. But it looks like not even new PR is working.

**Comment by sbc100** [comment]
I'm afraid I don't understand what is going on with circleci in this case.  I can try uploading this same PR and for you if you like (it should still mark you as the author of the commit I think).

**Comment by ravisumit33** [comment]
I came through https://discuss.circleci.com/t/could-not-find-a-usable-config-yml-you-may-have-revoked-the-circleci-oauth-app/46274.
Looks like contributors' PRs are failing. I think if you will upload it will not fail.

**Review by aheejin — APPROVED** [review]

**Comment by aheejin** [comment]
Hmm, I wondered if merging from a fork's `main` branch was a problem so I tested it (#18830)... and apparently that's not the issue. 🤷🏻

File: src/library_exceptions.js [hunk 1]
```diff
@@ -421,9 +421,11 @@ var LibraryExceptions = {
     //
     // Remove this JS function name, which is in the second line, from the stack
     // trace.
```
Comment (kripken) [inline src/library_exceptions.js:423]:
```suggestion
    // trace. Note that .stack does not yet exist in all browsers (see #18828).
```

**Comment by kripken** [comment]
I saw `Maintainers are allowed to edit this pull request.` so I tried to add a small text diff and commited that, hoping it makes CircleCI work... looks like it might have helped.

**Pull request merged**